src/containers/App/index.js

```diff
@@ -29,7 +29,7 @@ export const AppContainer = ({ requestCategoriesAction }) => {
 
   useEffect(() => {
     requestCategoriesAction();
-  }, []);
+  }, [requestCategoriesAction]);
```

Comment:
We need this, because `requestCategoriesAction` is a dependency of the `useEffect` hook. Even though the function signature won't change, it's best practice to list all dependencies unless limiting in any way.